core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -355,6 +358,41 @@ public void removeBlock(long sessionId, long blockId)
     mBlockStore.removeBlock(sessionId, blockId);
   }
 
+  public void freeWorker() throws IOException {
+    List<String> paths = new ArrayList<>();
+    if (Configuration.global().get(PropertyKey.WORKER_BLOCK_STORE_TYPE) == BlockStoreType.FILE) {
+      int tierCount = Configuration.global().getInt(PropertyKey.WORKER_TIERED_STORE_LEVELS);
+      for (int i = 0; i < tierCount; i++) {
+        paths.addAll(Configuration.global().getList(PropertyKey
+                .Template.WORKER_TIERED_STORE_LEVEL_DIRS_PATH.format(i)));
+      }
+    } else {
```

Comment:
add an explicit check against `PropertyKey.WORKER_BLOCK_STORE_TYPE == BlockStoreType.PAGE`, so that in the future when a third block store option is added, this will fail loudly instead of silently deleting the wrong directory.